CHANGES.md

```diff
@@ -24,6 +24,7 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 * Metronome supports file based secrets
 * Metronome supports hybrid cloud
 
+* Expose Public IP (DCOS_OSS-4514)
```

Comment:
I don't want to block the PR on that; but I really cannot understand this (surely beautiful) feature based on these three words :D

Reply:
Would you expect to see some more information at the CHANGES.md? Or would like this PR to have more context to it by linking more information about the feature being exposed?

Reply:
Unfortunately, that's how the scoping/design document was named :shrug: